### DISCUSSION TITLE
I failed many times to apply for a student bag by using the online report of xuexin.com to verify my student identity

### DISCUSSION THREAD

**Discussion by 2144291529** [community/community#20089] [GitHub Education]
Using Xuexin online report to verify my student identity, I failed to apply for a student bag many times. Why is this? I really need a GitHub student bag. At the same time, I am really a sophomore student

**Comment by isCopyman** [comment]
me too

**Reply by 2144291529** [reply]
If you have a good way please let me know

**Reply by paudelsamir** [reply]
I'd suggest reaching out to GitHub Education support directly or trying another method to verify your student status. Maybe using an alternative verification service could work better.
